lms/views/basic_lti_launch.py

```diff
@@ -131,6 +132,10 @@ def db_configured_basic_lti_launch(self):
         in the LMS and passing it back to us in each launch request. Instead we
         retrieve the document URL from the DB and pass it to Via.
         """
+        # Configure front-end grading if feature is enabled
+        if self.request.feature("blackboard_grading"):
+            frontend_app.configure_grading(self.request, self.context.js_config)
```

Comment:
You could probably just pass the request here and let `frontend_app.configure_grading()` extract `request.context.js_config` itself. As things stand the view knows one thing about how to configure grading: where the config dict that needs to be modified comes from. But the helper knows everything else about how to configure grading.

`frontend_app` might work better as a service or resource because those already have access to the request (from Pyramid) so you wouldn't have to pass the request into it. But I think it's fine as a helper.

Reply:
> You could probably just pass the request here and let frontend_app.configure_grading() extract request.context.js_config itself.

I may end up doing that in a subsequent PR. I did it on purpose in the sense to make it easier to read the views without having to go look at the helper—you can conjecture more quickly that it's going to mess with `js_config` specifically. I'll consider this!

Reply:
> frontend_app might work better as a service or resource because those already have access to the request (from Pyramid) so you wouldn't have to pass the request into it. But I think it's fine as a helper.

Yeah, I may end up moving it if it feels awkward at any point.